app/experimenter/nimbus-ui/src/components/PageEditBranches/index.test.tsx

```diff
@@ -3,20 +3,23 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React from "react";
-import { screen } from "@testing-library/react";
+import { screen, waitFor } from "@testing-library/react";
 import PageEditBranches from ".";
 import { renderWithRouter } from "../../lib/test-utils";
 import { MockedCache, mockExperimentQuery } from "../../lib/mocks";
 
-const { mock } = mockExperimentQuery();
-
 describe("PageEditBranches", () => {
-  it("renders as expected", () => {
+  it("renders with experiment data", async () => {
+    const { mock } = mockExperimentQuery();
```

Comment:
So @jodyheavener, this is where I was working on tweaking `mockExperimentQuery` to get this to be nice (taking in a param to return a different mock/data), but the tweaks I tried couldn't get me past the `loading` state. Maybe it'll be more obvious with fresh eyes in the morning.